cmake/tests.cmake

```diff
@@ -0,0 +1,30 @@
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(CMOCKA REQUIRED cmocka)
+
+function(add_test_executable test_name)
```

Comment:
Nice, this a really cool solution for automating all the symbols that need to be wrapped for a test!